### ISSUE TITLE
getting transcript from multiple video ids?

### ISSUE THREAD

**Issue by helmimuharram** [jdepoix/youtube-transcript-api#121]
Hi I'm using windows at the moment and I used the lates version of the API. I just wonder how can I getting transcript from multiple video ids at the same time?

Currently I'm using the same code from https://stackoverflow.com/questions/42867732/getting-youtube-auto-transcript-from-api (the answer from user2585501) but for some reason the code only fetch the transcript of the last video id from the list.

Any tips for the reason/ how to solve? @jdepoix 

Thanks in advance!

**Comment by jdepoix** [comment]
Hi @helmimuharram, could you please share the exact code your using and error message you are getting doing so?

**Comment by helmimuharram** [comment]
`from youtube_transcript_api import YouTubeTranscriptApi

#srt = YouTubeTranscriptApi.get_transcript(video_id)

videoListName = "youtubeVideoIDlist.txt"
with open(videoListName) as f:
    video_ids = f.read().splitlines()

transcript_list, unretrievable_videos = YouTubeTranscriptApi.get_transcripts(video_ids, continue_after_error=True)

for video_id in video_ids:

    if video_id in transcript_list.keys():

        print("\nvideo_id = ", video_id)
        #print(transcript)

        srt = transcript_list.get(video_id)

        text_list = []
        for i in srt:
            text_list.append(i['text'])

        text = ' '.join(text_list)
        print(text)`

this is the code that I used. I'm not getting error or anything. But the result is not kinda what I expected. 

I thought I could get transcripts from multiple videos by looping from the list I have. But, it turns out I only get 1 transcript from 1 videos only. Is there a way to get multiple transcript from multiple videos at the same time? Thanks for the response @jdepoix

**Comment by jdepoix** [comment]
The easiest way to get multiple transcripts probably is `YouTubeTranscriptApi.get_transcripts(["video_id1", "video_id2", "..."])`. [Please read the README for more detailed information on how to use this module](https://github.com/jdepoix/youtube-transcript-api#api).

**Comment by helmimuharram** [comment]
wow, you really HAVE TO close this instantly huh. Well for your information: your solution doesn't work and if the features for multiple downloads didn't exist yet, maybe can you at least count it as a suggestion for the next update of this API.

**Comment by jdepoix** [comment]
I think you are kind of missing the point of GitHub issues. This should be used to report bugs in the repository, or discuss possible features/direction the repository should move in to, not so explain to you why some code you copied of stackoverflow doesn't work. The reason your code is not working as you expect it to does not have anything to do with this module. As I explained to you (although it is already clearly documented in the README), there is a simple way to get multiple transcripts. I am not going to explain to you how to read video IDs from a file etc. That's python basics. You can go to stackoverflow for that. This is not free python support! 
I work a full time job and do my best to support this repository in the limited free time I have so people can benefit from it (for free!). Therefore, I have to make sure to close issues as quickly as I can so that the issues count doesn't grow to an unmanageable number, and I have to quickly identify issues which are reporting an actual bug in this repository so that I can dedicate my time to fixing those. Please manage your expectations when using something for free!

**Comment by helmimuharram** [comment]
### here let me rephrase it for you:

> I'm sorry, but this place is only for reporting bugs or discussing new features in the repository. If you have a problem with some code from StackOverflow, then discuss it over there and not here. _**I'm sorry that I close your inquiries too early without explaining much**_ and _**not with the answer you expecting from this "free" repository**_. 
> 
> I might consider your suggestion for future version of the repository. **_Thanks!_** 

and for those of you who come here with the question same as mine (from google or whatever) and can't find the answer, the code is the same as what I paste from the StackOverflow, but just add this at the end:
`with open(video_id + '.txt', 'w', encoding='utf-8') as txt_file:
            txt_file.write(text)`

_I manage to find the solution before this thread closed but you do you I guess?_

**Comment by jdepoix** [comment]
### here let me rephrase it for you:
> I'm sorry, I don't quite understand why this got closed. Would you mind explaining, as my problem still remains?

Don't expect people to take the time to choose nice words for you while you don't bother to do so yourself.
